client/src/components/ResultRow.js

```diff
@@ -31,9 +31,9 @@ class ResultRow extends Component {
     let arrow = '>';
     let originalPosting = '';
     let applyButton = '';
-    let { location } = this.props.result;
+    let location = this.props.result.city + ", " + this.props.result.state + " " + this.props.result.country;
     if (this.state.selected) {
-      location = ''; // Hide things that shouldnt be shown when selected, show things that should be shown
+      location = ''; // Hide things that shouldnt be shown whenconstructor(title, link, description, excerpt, city, state, country, latitude, longitude, companyName, minSalary, startDate) selected, show things that should be shown
```

Comment:
Is the constructor supposed to be commented out?

Reply:
Also the database names are not camelCase, they are snake_case. Check out the schema [here](https://github.com/ubclaunchpad/Internado/wiki/Proposed-Database-Schema) for all the details. I'm not sure whether the properties of ResultData should be snake case as well, but if not, they need to be translated somewhere.